hawkular-dmr-client/src/main/java/org/hawkular/dmrclient/DeploymentJBossASClient.java

```diff
@@ -63,6 +68,21 @@ private void enableDisableDeployment(String name, boolean enable) throws Excepti
         return; // everything is OK
     }
 
+    public void restartDeployment(String name, Set<String> serverGroups) throws Exception {
+        DeploymentManager dm = DeploymentManager.Factory.create(getModelControllerClient());
+        DeploymentResult result;
+
+        SimpleDeploymentDescription sdd = SimpleDeploymentDescription.of(name);
+        sdd.addServerGroups(serverGroups);
```

Comment:
what if serverGroups is null? Does the API accept that? Or might be best to do:

sdd.addServerGroups(serverGroups != null ? serverGroups : Collections.emptySet());